### PR TITLE
Use ch profile

### PR DESCRIPTION
**Pull request by easbar**

### CODE REVIEW

File: matching-core/src/main/java/com/graphhopper/matching/MapMatching.java [hunk 1]
```diff
@@ -113,11 +113,10 @@ public MapMatching(GraphHopper graphHopper, AlgorithmOptions algoOptions) {
         RoutingAlgorithmFactory routingAlgorithmFactory = graphHopper.getAlgorithmFactory(hints);
         if (routingAlgorithmFactory instanceof PrepareContractionHierarchies) {
             ch = true;
-            // I want to use my own instance of FastestWeighting because it uses its own U-Turn-penalty,
-            // but I have to pass a _different_ instance of FastestWeighting to getGraph so it gives me the graph,
-            // (even though this is non-dangerous since it's only about virtual (split) edges (nothing to do with shortcuts),
-            // but it's _still_ a different one than the one which I have to pass to the router itself (see below).
-            routingGraph = graphHopper.getGraphHopperStorage().getCHGraph(((PrepareContractionHierarchies) routingAlgorithmFactory).getWeighting());
+            // I want to use my own instance of FastestWeighting because it uses its own heading penalty,
```
Comment (easbar) [inline matching-core/src/main/java/com/graphhopper/matching/MapMatching.java]: like this ? Are there multiple CH graphs ? If not we could just do `getCHGraph()`.

**Comment by easbar** [comment]
Since I have no doubt this does what it should I merged for now, still interested to hear your opinion on the change to the comment.

File: matching-core/src/main/java/com/graphhopper/matching/MapMatching.java (hunk 1 above)
Comment (karussell) [reply matching-core/src/main/java/com/graphhopper/matching/MapMatching.java]: Yes, there could be multiple CH graphs like for GH core.

**Comment by karussell** [comment]
Thanks!

**Pull request merged**